recipes/aws-cdi-sdk/all/patches/002-CMake.patch

```diff
@@ -33,6 +33,8 @@ index 9ca1679..193e136 100644
 +    set(ARCHIVE_DIRECTORY "${LIBRARY_DIRECTORY}")
 +endif()
 +
++set(CPP_STANDARD "11" CACHE STRING "Flag to upgrade the C++ standard used. The default is 11. The minimum is 11.")
```

Comment:
Maybe set it to https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html so conan can inject the users settings?